Oracle.md

```diff
@@ -0,0 +1,165 @@
+# Oracle specifications
+
+## Introduction
+
+For the purpose of these specifications, an event is a digital representation of a real world fact.
+An [oracle](./Introduction.md#Oracle) is an entity that commits to publishing a signature over a (number of) event outcome(s) ahead of time by releasing an [R-value](./Introduction.md#R-value) as well as necessary information for two parties to build a set of [CETs](./Introduction.md#Contract-Execution-Transaction-(CET)).
```

Comment:
nit: for consistency: "an R-value" -> "one (or more) R-value(s)"

Reply:
changed